lib/plugins/aws/package/compile/events/websockets/lib/authorizers.js

```diff
@@ -8,15 +8,20 @@ module.exports = {
       if (!event.authorizer) {
         return;
       }
-      const functionObj = this.serverless.service.getFunction(event.authorizer.name);
+
+      const isLocalFunction = !(event.authorizer.permission || '').startsWith('arn:');
```

Comment:
We support three different notations for `event.authorizer`:
- \<function arn\>
- \<function name\>
- Object with either `name` or `arn` property

So I think `isLocalFunction` should look more as:

```javascript
const isObject = require('type/object/is');
...
if (isObject(event.authorizer)) return event.authorizer.name
return !event.authorizer.startsWith('arn:')
```